tests/lookup/tests.py

```diff
@@ -909,3 +911,16 @@ def test_custom_lookup_none_rhs(self):
         field = query.model._meta.get_field('nulled_text_field')
         self.assertIsInstance(query.build_lookup(['isnull_none_rhs'], field, None), IsNullWithNoneAsRHS)
         self.assertTrue(Season.objects.filter(pk=season.pk, nulled_text_field__isnull_none_rhs=True))
+
+    def test_outerref_in_dates(self):
+        dates = Author.objects.filter(date__year__gte=OuterRef("year"))
```

Comment:
Use single quotes (unless a string contains a single quote) as described in [Python coding style](https://docs.djangoproject.com/en/dev/internals/contributing/writing-code/coding-style/#python-style).